code/modules/vehicles/mecha/equipment/tools/mining_tools.dm

```diff
@@ -179,21 +176,24 @@
 		else
 			target.investigate_log("has been gibbed by [src] (attached to [chassis]).", INVESTIGATE_DEATHS)
 			target.gib(DROP_ALL_REMAINS)
+		return
+
+	//drill makes a hole
+	var/def_zone = target.get_random_valid_zone(BODY_ZONE_CHEST)
+	var/obj/item/bodypart/target_part = target.get_bodypart(def_zone)
+	var/blocked = target.run_armor_check(def_zone, MELEE)
+	target.apply_damage(10, BRUTE, def_zone, blocked)
+
+	//blood splatters
+	var/splatter_dir = get_dir(chassis, target)
+	if(isalien(target))
+		new /obj/effect/temp_visual/dir_setting/bloodsplatter/xenosplatter(target.drop_location(), splatter_dir)
```

Comment:
Feel like we should delegate this to some proc somewhere but it seems to only be in two places (here and projectiles) so it's probably whatever